src/components/organisms/ChatSidebar/ChatSidebar.scss

```diff
@@ -12,13 +12,13 @@ $header-height: 45px;
   right: 0;
   width: $chat-sidebar-width;
   height: calc(100vh - 96px);
-  background-color: $saturated-black;
+  @include background-color($saturated-black);
```

Comment:
NOTE: function sets the needed CSS --variables for the descendant indicator border to preclude the direct parent's paint.